biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java

```diff
@@ -76,6 +78,46 @@
  */
 public class JPMSModuleInfoPlugin implements VerifierPlugin {
 
+	enum Access {
+		CLOSED(
+			0),
+		OPEN(ACC_OPEN),
+		SYNTHETIC(ACC_SYNTHETIC),
+		MANDATED(ACC_MANDATED);
+
+		public static Access parse(String input) {
+			switch (input) {
+				case "OPEN" :
+				case "open" :
+				case "0x0020" :
```

Comment:
Isn't this a bit restrictive? `0x20` wont work. You probably need two paths: string and numbers.

Reply:
k

Reply:
```
				default :
					int parsedValue = Integer.parseInt(input);
					return Arrays.stream(values())
						.filter(a -> a.getValue() == parsedValue)
						.findFirst()
						.orElse(CLOSED);

```
?

Reply:
I think you want `Integer.decode` which can handle hex strings and octal. But you should catch any exception from decode and map to an error.

Reply:
Also the orElse case should also be an error, no?